website/source/docs/guides/backup.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: "docs"
+page_title: "Cluster Backups"
+sidebar_current: "docs-guides-backups"
+description: |-
+  Consul provide the snapshot tool for backing up and restoring data. In this guide you will learn how to use both.
+---
+
+# Cluster Backups
+
+Creating server backups is an important step in production deployments. Backups provide a mechanism for the server to recover from an outage (network loss, operator error, or a corrupted data directory). All agents write to the `-data-dir` before commit. This directory persists the local agent’s state and - in the case of servers -  it also holds the Raft information.
+
+Consul provides the [snapshot](https://consul.io/docs/commands/snapshot.html) command which can be run using the CLI command or the API. The `snapshot` command saves the point-in-time snapshot of the state of the Consul servers which includes:
+
+* KV entries 
+* the service catalog
+* prepared queries
+* sessions 
+* ACLs
```

Comment:
This is not exhaustive and even if we made it so would get out of date, should we consider adding an element to the end of disclaimer that there is a bunch of other stuff too. For example: autopilot config, Connect CA config, intentions etc.

Reply:
@banks Can we identify the data that is most commonly saved? I am thinking we could have a smaller list of data that every environment is likely to have (KV entries, ACLs, service catalog). And then include a note that depending on the environment/configuration additional data will be saved (autopilot config, intentions, etc). Does that makes sense?